app/package.json

```diff
@@ -169,6 +169,7 @@
     "react-ace": "^6.4.0",
     "react-avatar-editor": "^11.0.4",
     "react-calendar": "^2.17.5",
+    "react-children-addons": "^1.0.9",
```

Comment:
We could probably remove this now/soon since the SearchPanel doesn't really need to count children anymore, and it didn't really work